tools/aif2pcm/main.c

```diff
@@ -469,12 +469,12 @@ do { \
 void aif2pcm(const char *aif_filename, const char *pcm_filename, bool compress)
 {
 	struct Bytes *aif = read_bytearray(aif_filename);
-	AifData aif_data = {0};
+	AifData aif_data = {0, 0, 0, 0, 0, 0, 0};
```

Comment:
it gives a "missing field 'x' initializer" error for every uninitialized field

Reply:
Is it actually an error? It's probably clang issuing spurious warnings as per usual (I've had clean compiles in GCC issue 10 screenfuls of warnings in clang).  
If it is a hard error, clang is broken (unless you're using `-Werror`) and you should file a bug. Otherwise, it might be a good idea to disable the diagnostic (it is spurious; the C standard is _very clear_ in that missing fields in struct/array initializers are zeroed)... or to just leave the extra zeroes for now.

Reply:
The Makefiles for all of the tools use `-Werror`

Reply:
Maybe there should be a clang Travis build to catch these things early, then. The flag here is `-Wno-missing-field-initializers`, if you want to take that route.

Reply:
do you have an argument for one over the other?

Reply:
I'd prefer the compiler switch, but I'm also against `-Wall -Wextra` and particularly `-Werror`, so maybe I'm not exactly a representative of the mean here.

Reply:
You can just disable the broken warning for clang; GCC ignores (or should ignore) unknown `-W` switches, so that's fine.

I'd also recommend reverting the two affected lines.